src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -781,7 +780,7 @@ BSONObj fillQueryServicePath(const std::vector<std::string>& servicePath)
         nullAdded = true;
       }
 
-      char path[SERVICE_NAME_MAX_LEN * 2];
+      char path[SERVICE_PATH_MAX_TOTAL * 2];
       slashEscape(servicePath[ix].c_str(), path, sizeof(path));
```

Comment:
The change seems to simple ...
From the functest we can see it works but ...

Did we not have a check of service path checking:
- Max total size: 110 (or something like that)
- Max length per 'path-component': 10 (later perhaps changed to 50)
- Max components: 10

What happened to those checks?
I mean, how come they didn't need a modification for this to work?